src/main/java/ui/MenuControl.java

```diff
@@ -32,12 +32,14 @@
     private final ScrollPane panelsScrollPane;
     private final UI ui;
     private final Preferences prefs;
+    private String openBoardName = "";
```

Comment:
Is there a need to keep track of this here as well, i.e. could this and the state in prefs ever go out of sync?